tests/PsrHttpClientClientExceptionInterface.phpt

```diff
@@ -1,7 +1,7 @@
 --TEST--
 Psr\Http\Client\ClientExceptionInterface
 --SKIPIF--
-<?php if( !extension_loaded('psr') ) die('skip '); ?>
+<?php if( !extension_loaded('psr') || PHP_MAJOR_VERSION < 7 ) die('skip '); ?>
```

Comment:
```suggestion
<?php include('skip_for_php5.inc'); ?>
```

```php
<?php
// skip_for_php5.inc

if (!extension_loaded("psr")) {
    print "skip The psr extension is not loaded";
} else if (PHP_MAJOR_VERSION < 7) {
    print "skip Due to version incompatibility";
}
?>
```

@flyinghail What do you think about something like this? Also as I know the `die` is not necessary. The `skip` word is enough.

Reply:
I think it's a good idea.

Reply:
@flyinghail Looks good to me. Could you change current tests?